client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -230,14 +230,17 @@ export class PluginsList extends Component {
 		this.props.removePluginStatuses( 'completed', 'error' );
 	}
 
-	doActionOverSelected( actionName, action ) {
+	doActionOverSelected(
+		actionName,
+		action,
+		selectedPlugins = this.props.plugins.filter( this.isSelected ) // only use selected sites
```

Comment:
Is there any reason to initialize `selectedPlugins` within the function declaration? I'd move it in the function scope for better readability.